### PR TITLE
Initial commit for 2.5.3 parsing

### PR DESCRIPTION
**Pull request by killerwife**
Will require a full opcode review and update object isnt parsing correctly anymore in the movement part. Likely something new added in shadowlands.

### CODE REVIEW

**Comment by Shauren** [comment]
probably spline changes

**Comment by Shauren** [comment]
Just checked 2.5.1 parser, its the same as shadowlands for units, later shadowlands versions only altered areatriggers but i dont think those exist in 2.5.x (structure does but are they ever created?)

**Comment by killerwife** [comment]
No they are not ever created in anything until something after wotlk

**Comment by Shauren** [comment]
Might be a 9.2 ptr change then

**Comment by killerwife** [comment]
Apparently 1.14 has had same issue (and its not yet on tc wpp, just ratkosrbs) and likely got rolled out to tbc now. I bet it will make its way to shadowlands too.

**Comment by killerwife** [comment]
@Shauren I fixed parsing of movement part, needs new update fields however and thats out of my ballpark.

File: WowPacketParserModule.V2_5_1_38707/Parsers/UpdateHandler.cs [hunk 1]
```diff
@@ -138,6 +138,14 @@ private static MovementInfo ReadMovementUpdateBlock(Packet packet, WowGuid guid,
                 packet.ResetBitReader();
                 packet.ReadPackedGuid128("MoverGUID", index);
 
+                if (ClientVersion.AddedInVersion(ClientVersionBuild.V2_5_3_41818))
+                {
+                    moveInfo.Flags = (MovementFlag)packet.ReadBitsE<V6_0_2_19033.Enums.MovementFlag>("Movement Flags", 32, index);
+                    moveInfo.FlagsExtra = (MovementFlagExtra)packet.ReadBitsE<V8_0_1_27101.Enums.MovementFlags2>("Extra Movement Flags", 32, index);
+
+                    packet.ReadUInt32("Unk1", index);
```
Comment (MaxtorCoder) [inline WowPacketParserModule.V2_5_1_38707/Parsers/UpdateHandler.cs:146]: In the 920 client this is an array of 3x uint32 and points to MovementFlags.

**Pull request merged**